apps/election-manager/src/components/ExportElectionBallotPackageModalButton.tsx

```diff
@@ -104,23 +104,33 @@ const ExportElectionBallotPackageModalButton: React.FC = () => {
   )
 
   // Callback to open the file dialog.
-  const openFileDialog = async () => {
+  const saveFileCallback = async (openDialog: boolean) => {
     if (state.type !== 'ArchiveBegin') {
       throw new Error(
-        `unexpected state '${state.type}' found during openFileDialog callback`
+        `unexpected state '${state.type}' found during saveFileCallback`
       )
     }
     try {
-      await state.archive.begin({
-        defaultPath: defaultFileName,
-        filters: [{ name: 'Archive Files', extensions: ['zip'] }],
-      })
+      const usbPathRaw = await getDevicePath()
+      const usbPath = usbPathRaw ? `${usbPathRaw}/` : ''
```

Comment:
When would this be falsy? That seems like a time to abort this action, right?

Reply:
I think this will just happen in some of the test environments, I'll see if I can fix those tests a better way though